src/cmd/cmd.cpp

```diff
@@ -294,7 +294,7 @@ void selectiveSyncFixup(OCC::SyncJournalDb *journal, const QStringList &newList)
     if (ok) {
         auto blackListSet = newList.toSet();
         auto changes = (oldBlackListSet - blackListSet) + (blackListSet - oldBlackListSet);
-        foreach (const auto &it, changes) {
+        for (const auto &it : qAsConst(changes)) {
```

Comment:
Note that the alternative would have been to declare `blackListSet` and `changes` as `const auto`. That'd work as well.

(I'm not asking you to change it if you don't want to, it's more of a "BTW" comment).